src/oscar/apps/dashboard/views.py

```diff
@@ -146,13 +134,32 @@ def get_stats(self):
         datetime_24hrs_ago = now() - timedelta(hours=24)
 
         orders = Order.objects.all()
+        alerts = StockAlert.objects.all()
+        baskets = Basket.objects.filter(status=Basket.OPEN)
+        customers = User.objects.filter(orders__isnull=False).distinct()
+        lines = Line.objects.filter()
+
+        user = self.request.user
+        if not user.is_staff:
```

Comment:
I think it's worth adding some tests to check that these filters return the expected results.